addons/account/models/account.py

```diff
@@ -714,16 +715,25 @@ def compute_all(self, price_unit, currency=None, quantity=1.0, product=None, par
             if tax.price_include:
                 total_excluded -= tax_amount
                 base -= tax_amount
+                tax_base -= tax_amount
+                tax_excluded -= tax_amount
             else:
                 total_included += tax_amount
+                tax_included += tax_amount
 
             if tax.include_base_amount:
                 base += tax_amount
 
+            if tax.amount_type in ['division']:
```

Comment:
I don't get the point. If you have a 10 % Percentage of Price Tax Included on 100 EUR, I would expect the base to be 90 and the tax to be 10. Why add the tax amount on the base?